src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -110,17 +90,25 @@ export class ActionMenu extends PureComponent<ActionMenuProps, State> {
         )}
       </div>
     );
-  };
+  }
 
-  private handleMenuGroupToggle = (group: string) => {
-    this.setState(({activeMenuGroup}) => ({
-      activeMenuGroup: activeMenuGroup ? undefined : group,
-    }));
-  };
+  function handleMenuGroupToggle(group: string) {
+    setActiveMenuGroup(activeMenuGroup ? undefined : group);
```

Comment:
This was a callback before but I don't think it needed to be. Any objections to removing it?

Reply:
No objections here.

Reply:
If this gets wrapped in a useCallback (which I think it should be so you don't trigger rerenders of MenuGroup because you pass in a new function every rerender) then you'll want to use the callback style notation so you don't end up creating a new function every time activeMenuGroup changes